projects/sbb-esta/angular-public/src/lib/checkbox/checkbox/checkbox.component.html

```diff
@@ -16,6 +16,7 @@
 
   <div class="sbb-checkbox-container">
     <sbb-icon-tick class="sbb-checkbox-checked"></sbb-icon-tick>
+    <div *ngIf="!checked" class="sbb-checkbox-indeterminate"></div>
```

Comment:
This I would like to avoid.
Indeterminate does not exist in public.
Is this maybe possible with pure css? (:content)
However, how can the consumer activate the indeterminate state?
I think extending the component might be the better approach here.

Reply:
I created a distinct checkbox component for business, which has an indeterminate state.

Please note, that I don't update the indeterminate state of the native element, however, I update ariaChecked. Do you think this is sufficient or should I update the native element as well?

Reply:
Please create a new template for the business variant, which forwards the indeterminate state to the native checkbox element (<input ... [indeterminate]="indeterminate" ...>).
indeterminate can probably be reduced to a simple property, instead of getter/setter.
You can also attach `@HostBinding('class.sbb-checkbox-indeterminate')` to the indeterminate property. `indeterminateClass` is unnecessary.